src/main/java/com/microsoft/sqlserver/jdbc/SQLServerConnection.java

```diff
@@ -4520,6 +4520,8 @@ final void processFedAuthInfo(TDSReader tdsReader, TDSTokenHandler tdsTokenHandl
 
         aadPrincipalSecret = "";
         activeConnectionProperties.remove(SQLServerDriverStringProperty.AAD_SECURE_PRINCIPAL_SECRET.toString());
+
+        activeConnectionProperties.remove(SQLServerDriverStringProperty.TRUST_STORE_PASSWORD.toString());
```

Comment:
Any specific reason to call it inside "processFedAuthInfo"?
Can you confirm this is called when there is no FedAuthInfo?

Reply:
you are right I put this in the same place as the hotfix, but this should b somewhere else....🤦‍♀️